assets/js/event-filtering.js

```diff
@@ -1,37 +1,36 @@
 $(function() {
-    var eventsElements = $(".event-tags span");
-    var eventFilterParent = $("#eventFilter");
-
-    // Check to see if the event elements exists.
-    if (!eventsElements.length) {
-        // Stop the script as there are no tags to filter.
-        return;
-    }
-    if (!eventFilterParent.length) {
-        // Stop the script because the event filter element does not exist.
-        return;
-    }
-
-    var tags = [];
-
     /**
      * This function creates a checkbox from a given text string.
      * It will set the label and the value to whatever the input string is.
      *
      * @param {string} text The label and value of the checkbox
      */
     function createCheckbox(text) {
+        var filterValue = getQueryVariable("filter");
+
         var container = document.createElement("div");
-        container.className = "my-2";
+        container.className = "my-2 uppercase flex items-center";
 
         var checkbox = document.createElement("input");
         checkbox.id = "checkbox-" + text;
         checkbox.type = "checkbox";
-        checkbox.className = "mx-2 cursor-pointer";
-        checkbox.value = text;
+        checkbox.className = "mr-2 cursor-pointer";
+        checkbox.value = text.toLowerCase();
+
+        // If the filter query parameter is available use it to determine
+        // what checkboxes should initially be checked. If the query param
+        // is not set, check all the boxes.
+        if (filterValue !== undefined) {
+            var shouldBeChecked = filterValue.split(",").indexOf(checkbox.value) > -1;
```

Comment:
Nit: `getQueryVariable` doesn't lowercase the value, which you were doing previously, so if you want to maintain that behavior, you'll have to do something like:

```suggestion
            var shouldBeChecked = filterValue.toLowerCase().split(",").indexOf(checkbox.value) > -1;
```